examples/tcp-retransmit.bpf.c

```diff
@@ -0,0 +1,88 @@
+#include <vmlinux.h>
+#include <bpf/bpf_tracing.h>
+#include "maps.bpf.h"
+#include <bpf/bpf_core_read.h>
+
+#define MAX_ENTRIES 8192
+#define RETRANSMIT  1
+#define TLP         2
+
+#define AF_INET		2
+#define AF_INET6	10
+
+struct ipv4_key_t {
+	u32 saddr;
+	u32 daddr;
+	__u16 sport;
+	__be16 dport;
+    int type;
```

Comment:
`int` is ambiguous, please use a properly sized type like `u8`

Reply:
Okay, I changed the type to u32